packages/fast-usdc/src/exos/advancer.js

```diff
@@ -103,6 +105,10 @@ export const stateShape = harden({
 });
 
 /**
+ * Advancer subscribes (using handleTransactionEvent) to events published by the
+ * transaction feed. When notified of an appropriate opportunity, it is
```

Comment:
nb: this could be helpful for navigation:

```suggestion
 * {@link TransactionFeedKit}. When notified of an appropriate opportunity, it is
```

With this up top:
```js
 * @import {TransactionFeedKit} from './transaction-feed.js';
```

Reply:
thanks.